api/src/main/java/org/openmrs/module/openconceptlab/Subscription.java

```diff
@@ -86,11 +94,11 @@ public boolean isManual() {
 		return (days == null && hours == null && minutes == null) || (days == 0 && hours == 0 && minutes == 0);
 	}
 
-	public boolean isSubscribedToSnapshot() {
+	public Boolean getSubscribedToSnapshot() {
```

Comment:
Yes you did and I agree with it. But like i responded to you, I'm getting some validation error in webservices abstraction classes when using boolean